File: lib/src/main/java/uk/iatom/rfm69/registers/values/RssiThresh.java
package uk.iatom.rfm69.registers.values; import uk.iatom.rfm69.registers.IRegisterValue;

public enum RssiThresh implements IRegisterValue {
    

    // RegRssiThresh),
    VALUE(0xE4); // Default),;

    private byte val;
    
    RssiThresh(int val) {
        this.val = (byte) val;
    }

    @Override public byte val() {
        return val;
    }
}

File: lib/src/main/java/uk/iatom/rfm69/registers/values/RssiThresh.kt
package uk.iatom.rfm69.registers.values

import uk.iatom.rfm69.registers.IRegisterValue

enum class RssiThresh(`val`: Int): IRegisterValue { // RegRssiThresh),
VALUE(0xE4);


    // Default),;
    private val `val`: Byte

    init {
        this.`val` = `val`.toByte()
    }

    override fun `val`(): Byte {
        return `val`
    }
}
